pyproject.toml

```diff
@@ -43,8 +43,8 @@ python = "^3.7"
 
 dataclasses = {version = "^0.8", python = "~3.6.2"}
 fastapi = "^0.89"
-pydantic = "^1.0,<2.0"
-sqlalchemy =  { version = "^1.4,<2.0", optional = true }
+pydantic = ">=1.10,<2.0"
```

Comment:
Theres an error in 1.9 in python 3.11, this bumps to a compatible version.

Reply:
You are right, however, out of backward compatibility, I don't want to force anyone to update (if for some reason they are using older pydantic). I thought I fixed it in master branch, but if it still causes issues I'll fix it as well